sympy/physics/units/definitions.py

```diff
@@ -53,7 +53,7 @@
 # gram; used to define its prefixed units
 g = gram = grams = Quantity("gram", abbrev="g")
 gram.set_dimension(mass)
-gram.set_scale_factor(One)
+gram.set_scale_factor(Rational(1, 1000))
 
 # NOTE: the `kilogram` has scale factor 1000. In SI, kg is a base unit, but
```

Comment:
Does this note still apply?

Reply:
According to [this comment](https://github.com/sympy/sympy/pull/15664#issuecomment-449357217) it seems to apply.